xslt/mathml-to-text/mathml-to-placeholder.xsl

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
```

Comment:
Duplikat av den i epub-to-dtbook-audio? Bør bare ligge ett sted. Kan godt ligge her.

Reply:
Både ja og nei, jeg har lagt til en ekstra bool i den nå for å generere bilder - vet ikke hvordan denne kan settes fra python så må derfor være en fil per true|false. Du trenger f.eks. ikke generere bilder for audio.

Reply:
Skal vi lage en shared folder først som sist? @josteinaj

Reply:
Ja, det lagde du vel (eller Per?) allerede? `mathml-to-text`-mappen?

Reply:
Ja, men den er vel koblet til mathml-to-text prosessen?

Reply:
Den kan brukes fra hvor som helst, det er vel ingen pipeline som heter "mathml-to-text", så da er det i praksis en mappe med utility-ting.